qml/pages/Exhibitors/Exhibitors.qml

```diff
@@ -8,6 +8,9 @@ WebView {
     height: window.height
     url: "https://www.socallinuxexpo.org/scale/20x/exhibitors"
 
+    // Used to move the WebView when the drawer is opened due to https://doc.qt.io/qt-5/qtwebview-index.html#Limitations
+    x: window.width * window.drawerPosition
```

Comment:
* or + ?

Reply:
window.drawerPosition is a real type that measures from 0 to 1 with 0 being a closed drawer and 1 being a fully opened drawer.

Reply:
I got the name from [this drawer doc page](https://doc.qt.io/qt-5/qml-qtquick-controls2-drawer.html#position-prop)

Reply:
Ohhh! Carry on :)